app/models/user.rb

```diff
@@ -52,6 +53,6 @@ def requested_match(user)
   end
 
   def matches
-    outgoing_matches.append(incoming_matches).order(match_at: :asc)
+    Match.where("challenger_id = :user_id OR defender_id = :user_id", user_id: id).order(match_at: :desc)
```

Comment:
I end up calling it in a bunch of controllers. I'm not sure if they could access the method if its private